modules/virt-example-ethernet-nncp.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * virt/node_network/virt-updating-node-network-config.adoc
+
+[id="virt-example-ethernet-nncp_{context}"]
+= Example: Ethernet interface NodeNetworkConfigurationPolicy
+
+Configure an ethernet interface on nodes in the cluster by applying a `NodeNetworkConfigurationPolicy` manifest to the cluster.
+
+The following YAML file is an example of a manifest for an ethernet interface.
+It includes samples values that you must replace with your own information.
+
+[source,yaml]
+----
+
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: eth1-policy <1>
+spec:
+  nodeSelector: <2>
+    kubernetes.io/hostname: <node01> <3>
+  desiredState:
+    interfaces:
+    - name: eth1 <4>
+      description: Configuring eth1 on node01 <5>
+      type: ethernet <6>
+      state: up <7>
+      ipv4:
+        dhcp: true <8>
+        enabled: true <9>
+----
+<1> Name of the Policy.
+<2> Optional. If you do not include the `nodeSelector`, the Policy applies to all nodes in the cluster.
```

Comment:
I think the OCP docs moved from `Optional.` to `Optional:`

Reply:
In that case I've updated all the instances in this assembly